dimod/generators/fcl.py

```diff
@@ -144,6 +155,17 @@ def frustrated_loop(graph: GraphLike,
         raise RuntimeError(f"number of found cycles, {good_cycles}, is below " 
                            f"requested {num_cycles}")
 
+    if planted_solution is not None:
+        warnings.warn("planted_solution is deprecated; use spin-reversal transforms: "
+                      "(1) bqm.flip_variable() for superficial reorientation of "
+                      "spin labels to match desired planting or "
+                      "(2) SpinReversalComposite for presentation of a randomized "
+                      "planted solution to some solver.",
+                      DeprecationWarning)
+        # A spin-reversal transform for a BQM with zero linear biases
+        for e in bqm.quadratic:
+            bqm.quadratic[e] = bqm.quadratic[e]*planted_solution[e[0]]*planted_solution[e[1]]
```

Comment:
doesn't really matter, but there is a slight performance benefit to using `bqm.set_quadratic()` rather than `bqm.quadratic[]`